packages/backend/src/utils/inferenceUtils.ts

```diff
@@ -115,6 +115,14 @@ export function generateContainerCreateOptions(
     throw new Error('The model info file provided is undefined');
   }
 
+  const envs: string[] = [`MODEL_PATH=/models/${modelInfo.file.file}`, 'HOST=0.0.0.0', 'PORT=8000'];
+  if (modelInfo.properties) {
+    Object.entries(modelInfo.properties).forEach(([key, value]) => {
+      const formatedKey = key.replace(/[A-Z]/g, m => `_${m}`);
+      envs.push(`${formatedKey}=${value}`);
+    });
```

Comment:
looks like formatedKey has a typo (missing two t) but it's not really the matter
AFAIK it's missing the toUppercase
FYI you can have an array from entries, so no need to have a forEach loop as you can from transformation have an array and then you can destructure the array with `...`

so something like (not tested)
```ts
const additionalEnvs = Array.from(Object.entries(modelInfo.attributes)).map((e) => `${e[0].replace(/[A-Z]/g, m => `_${m}`).toUpperCase()}=${e[1]}`)
envs.push(...additionalEnvs);
```

Reply:
````ts
const additionalEnvs = Array.from(Object.entries(modelInfo.attributes)).map((e) => `${e[0].replace(/[A-Z]/g, m => `_${m}`).toUpperCase()}=${e[1]}`)
envs.push(...additionalEnvs);
````

I am thinking that is hard to read and understand for an external reader

Reply:
well you can extract key and value and assemble them in the map function but it's still using stream pattern and not loop pattern.

Reply:
thanks